runtime/onert/backend/open_cl/AccessType.h

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Most of changes on this commit came from Tensorflow 2.3 branch.
Would you please leave a note of the origin reference commit number from Tensorflow git?
As there's a lot of diffs per branches and per versions, keep tracking of those changes are required